src/main/webapp/index.html

```diff
@@ -134,7 +134,7 @@ <h2 class="panel-row__header u-align-with-first-panel">Build your plan:</h2>
         <p> Your unread emails contain <b id="word-count"></b> words
         <p>Average reply: <b id="average-reply"></b> words
         <p>Time needed: <b id="time-needed"></b> minutes
-        <p> Click to schedule
+        <p id = "message-event">
```

Comment:
nit: Remove the spaces so that it is `id="message-event"`

Reply:
`Resolved`